test/e2e/federated-service.go

```diff
@@ -144,7 +143,7 @@ var _ = framework.KubeDescribe("Service [Feature:Federation]", func() {
 	It("should be able to discover a federated service", func() {
 		framework.SkipUnlessFederated(f.Client)
 
-		createService(f.FederationClientset, clusterClientSets, f.Namespace.Name)
+		createService(f.Clientset_1_3, clusterClientSets, f.Namespace.Name)
```

Comment:
As discussed this wont work.
We will need to add a versioned FederationClientset to the framework